### PR TITLE
[BUG-73456] adding superset and move broker to master group

### PR DESCRIPTION
**Pull request by b-slim**
In This PR i have added superset UI as part of the druid bi cluster.
Also since it is a master need to be present on the ETL cluster otherwise install can fail.
Injecting email, admin user name and password. The idea is to use the same password and user name for ambari.

---
This change is [Reviewable](https://reviewable.io/reviews/sequenceiq/cloudbreak/2250)

### CODE REVIEW

**Comment by b-slim** [comment]
@martonsereg can you please point me to the code where i can inject the password and username like for zepplin ?

Thanks

**Comment by b-slim** [comment]
@martonsereg can you please take a look on this. Thanks

**Comment by b-slim** [comment]
> cluster.getEmailTo() can be null, if someone doesn't want email updates for the cluster

@martonsereg i thought the user need to provide an email before the creation of the could controller ? is that correct ? if it is true can we default to that one if `cluster.getEmailTo` is `null` ?

> cluster.getOwner() is a generated UUID, that's only for internal use in Cloudbreak.

is it possible to pick the ambari user name provided by the user when he is launching  the Amabri cluster ?

**Comment by doktoric** [comment]
@b-slim the emailto field is only available if the user provide email information for us and this is not always the case. 
I suggest to use the email address of the user who created the resource.
You can find an example here: https://github.com/sequenceiq/cloudbreak/blob/master/core/src/main/java/com/sequenceiq/cloudbreak/converter/CloudbreakUsageToJsonConverter.java#L37
The username of the **cbuser** object is the mail address of the creator.
br,
R

**Comment by b-slim** [comment]
@doktoric  thanks for the tip ! please checkout the fix.

File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/blueprint/DruidSupersetConfigProvider.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  public String addToBlueprint(Stack stack, String blueprintText) {
+    if (blueprintProcessor.componentExistsInBlueprint("DRUID_SUPERSET", blueprintText)) {
+      LOGGER.info("Druid Superset exists in Blueprint");
+      List<BlueprintConfigurationEntry> configs = getConfigs(stack.getCluster());
+      blueprintText = blueprintProcessor.addConfigEntries(blueprintText, configs, false);
+    }
+    return blueprintText;
+  }
+
+  private List<BlueprintConfigurationEntry> getConfigs(Cluster cluster) {
```
Comment (b-slim) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/blueprint/DruidSupersetConfigProvider.java]: @nishantmonu51 we could, is it better than the default one ?

**Pull request merged**